src/MapViewController.swift

```diff
@@ -446,6 +451,9 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
     findMeButton.setBackgroundImage(UIImage(named: "ic_find_me_pressed"), for: [.selected])
     findMeButton.backgroundColor = UIColor.white
     findMeButton.autoresizingMask = [.flexibleTopMargin, .flexibleLeftMargin]
+    let viewRect = mapView.bounds
```

Comment:
I was focused on removing it from `viewWillAppear` since it doesnt need to be added every time the view appears but youre right, this isn't the place to add it. I've moved calling of this method from the initializer into `viewDidLoad`